app/scripts/filters/resources.js

```diff
@@ -120,16 +120,29 @@ angular.module('openshiftConsole')
       });
       return nameCount;
     }
-    return function (resource, projects){
+    function escapeLessThanGreaterThan(str) {
+      return str.
+        replace(/</g, '&lt;').
+        replace(/>/g, '&gt;');
+    }
```

Comment:
> Are we sure there isn't a problem with other content?

No, but wouldn't this be a much larger issue for anyone using ui-select?  Is `ng-bind-html` not protecting us?